automation/scripts/compile_protobuf320_for_mac_arm64.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```

Comment:
1. I tried running this twice - it didn't work for me:
```
  ERROR: Failed building wheel for protobuf
  Running setup.py clean for protobuf
  error: subprocess-exited-with-error

  × python setup.py clean did not run successfully.
  │ exit code: 1
  ╰─> [14 lines of output]
      /private/var/folders/lg/ql9f9tm107117xqt2vcxz2bc0000gp/T/pip-install-lmgck3iv/protobuf_707bb7c1ad7143b9ad1f594d921e9fdb/setup.py:11: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
        import pkg_resources
      /Users/Jonathan_Daniel/projects/mlrun/venv/lib/python3.9/site-packages/setuptools/dist.py:678: SetuptoolsDeprecationWarning: The namespace_packages parameter is deprecated.
      !!

              ********************************************************************************
              Please replace its usage with implicit namespaces (PEP 420).

              See https://setuptools.pypa.io/en/latest/references/keywords.html#keyword-namespace-packages for details.
              ********************************************************************************

      !!
        ep.load()(self, ep.name, value)
      invalid command name '"--cpp_implementation"'
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed cleaning build dir for protobuf
Failed to build protobuf
ERROR: ERROR: Failed to build installable wheels for some pyproject.toml based projects (protobuf
```
2. That's quite a long script. I see that you added some references, but I think it will be hard to maintain.